talk/owt/sdk/ic/icmanager.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OWT_IC_ICMANAGER_H_
+#define OWT_IC_ICMANAGER_H_
+
+#include "talk/owt/sdk/include/cpp/owt/ic/icmanagerinterface.h"
+
+#include <memory>
+
+namespace InferenceEngine {
+class Core;
+}
+
+namespace owt {
+namespace ic {
+
+class ICManager : public ICManagerInterface {
+ public:
+  ICManager();
+
+  bool InitializeInferenceEngineCore(
+      const std::string& plugins_xml_path) override;
+
+  std::shared_ptr<owt::base::VideoFramePostProcessor> CreatePostProcessor(
+      ICPlugin plugin) override;
+
+ protected:
+  std::shared_ptr<InferenceEngine::Core> core;
+};
+
+}  // namespace ic
+}  // namespace owt
+
+#ifdef WEBRTC_WIN
```

Comment:
move OWT_IC_EXPORT/OWT_IC_IMPL def to a separate file.

Reply:
What should the file name be? `icapi.h`?